src/Security/Group.php

```diff
@@ -234,8 +234,8 @@ class_exists(SecurityAdmin::class)
                     ) . '<br />' .
                     sprintf(
                         '<a href="%s" class="add-role">%s</a>',
-                        SecurityAdmin::singleton()->Link('show/root#Root_Roles'),
-                        _t('SilverStripe\\Security\\Group.RolesAddEditLink', 'Manage roles')
+                        SecurityAdmin::singleton()->Link('roles'),
+                        _t(__CLASS__ . '.RolesAddEditLink', 'Manage roles')
```

Comment:
Changing `'SilverStripe\\Security\\Group.RolesAddEditLink'` to `__CLASS__ . '.RolesAddEditLink'` is an unrelated refactoring - figured I should tidy that up while I'm in here.